rust-symcrypt/Cargo.toml

```diff
@@ -22,7 +22,7 @@ lazy_static = "1.4.0"
 default = []
 md5 = []
 sha1 = []

```

Comment:
these should be `weak-sha1` and `weak-md5`. You were correct with changing the `weak-crypto` feature, but right now when you apply `weak-crypto` you are enabling `md5` and `sha1` which is not implemented anywhere in the code